packages/evolution-frontend/tests/ui-testing/testHelpers.ts

```diff
@@ -585,6 +626,26 @@ export const inputDatePickerTest = ({ context, path, value }: PathAndValue & Com
     });
 };
 
+/**
+ * Input a date that is invalid (such as being in the future) and check that it gets autocorrected to a different value.
```

Comment:
It does not really check that it is autocorrected to a different value, it just makes sure the value is not the requested one. What is the actual expected value? Do we know? Or is it empty? You can update the doc here

Reply:
It seems to depend on the situation and context. For example, today the latest valid date is 01/08/2024. If I type 02/08/2024 in an empty box, it gets corrected to 02/08/2020. However, if I first input 01/08/2024, and then try to change it to 02/08/2024, it changes back to 01/08/2024.